setup.cfg

```diff
@@ -29,12 +29,16 @@ package_dir=
 packages=find:
 install_requires =
     snakemake >= 6.8.1
-    fluidsim-core >= 0.5.0
+    # fluidsim-core >= 0.5.1 (unreleased)
+    fluidsim-core @ hg+https://foss.heptapod.net/fluiddyn/fluidsim#egg=fluidsim-core&subdirectory=lib
```

Comment:
I still need to release fluidsim-core >= 0.5.1 and fluiddyn >= 0.4.0 before merging. So these lines will be removed before merging.